packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -36,6 +38,27 @@ export async function getCustomTabsSupportingBrowsersAsync(): Promise<CustomTabs
   return ExponentWebBrowser.getCustomTabsSupportingBrowsersAsync();
 }
 
+export async function warmUpAsync(browserPackage?: string) {
+  if (!ExponentWebBrowser.warmUpAsync) {
+    throw new UnavailabilityError('WebBrowser', 'warmUpAsync');
+  }
+  return ExponentWebBrowser.warmUpAsync(browserPackage);
+}
+
+export async function mayInitWithUrlAsync(url: string, browserPackage?: string) {
+  if (!ExponentWebBrowser.mayInitWithUrlAsync) {
+    throw new UnavailabilityError('WebBrowser', 'mayInitWithUrlAsync');
+  }
+  return ExponentWebBrowser.mayInitWithUrlAsync(url, browserPackage);
+}
+
+export async function coolDownAsync(browserPackage?: string) {
+  if (!ExponentWebBrowser.coolDownAsync) {
+    throw new UnavailabilityError('WebBrowser', 'coolDownAsync');
+  }
+  return ExponentWebBrowser.coolDownAsync(browserPackage);
```

Comment:
I'm wondering… 🤔 should we also implement those methods on iOS (for the mocks to be available)? Methods would do nothing and the user wouldn't need to `if (platform)`. cc @tsapeta

Reply:
Yeah, I agree with @sjchmiela 👍